Common/Product/ReplWindow/Guids.cs

```diff
@@ -4,22 +4,14 @@
 
 using System;
 
-#if NTVS_FEATURE_INTERACTIVEWINDOW
```

Comment:
This is just some clean up, since we only build the interactive window for NTVS. And this was not making the code easier to read.